dev-embedded/usbprog/usbprog-0.3.0_p20140828-r2.ebuild

```
@@ -1,15 +1,15 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit cmake readme.gentoo-r1
 
-REV="293d559bac55"
+REV="293d559bac55f7c7130ea2769c703c68a19d62c2"
 
 DESCRIPTION="flashtool for the multi purpose programming adapter usbprog"
-HOMEPAGE="http://www.aaabbb.de/FirmwareUsbprog/FirmwareUsbprog_en.php https://bitbucket.org/bwalle/usbprog-tools/src/master/"
-SRC_URI="https://bitbucket.org/bwalle/usbprog-tools/get/${REV}.tar.bz2 -> ${P}.tar.bz2"
+HOMEPAGE="https://github.com/bwalle/usbprog-tools http://www.aaabbb.de/FirmwareUsbprog/FirmwareUsbprog_en.php"
```

```suggestion
HOMEPAGE="https://github.com/bwalle/usbprog-tools https://www.aaabbb.de/FirmwareUsbprog/FirmwareUsbprog_en.php"
```
I can do this while merging, just pointing it out